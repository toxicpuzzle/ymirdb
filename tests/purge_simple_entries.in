SET c 7
SET b 3 5 1
SET a 1 2 3
SET d 1 1 1
LIST ENTRIES
SNAPSHOT
SET a 2 3 4
SNAPSHOT
PUSH a 1 2 3
SNAPSHOT
PURGE c
LIST ENTRIES
CHECKOUT 2
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
bye