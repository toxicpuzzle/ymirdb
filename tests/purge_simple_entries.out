> ok

> ok

> ok

> ok

> d [1 1 1]
a [1 2 3]
b [3 5 1]
c [7]

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> d [1 1 1]
a [3 2 1 2 3 4]
b [3 5 1]

> ok

> d [1 1 1]
a [2 3 4]
b [3 5 1]

> ok

> d [1 1 1]
a [1 2 3]
b [3 5 1]

> bye
